internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1221,6 +1221,15 @@ func (r *ClusterDeploymentsReconciler) TransformClusterToDay2(
 	if err != nil {
 		log.WithError(err).Errorf("failed to transform cluster %s into day2 cluster", cluster.ID.String())
 	}
+	if clusterInstall.Spec.IgnitionEndpoint == nil {
+		// Set custom Ignition endpoint so that day2 workers would join using HTTPS endpoint
+		// ensureMCSCert would add ignition override with MCS secret
+		ignitionEndpoint := fmt.Sprintf("https://api-int.%s.%s:22623/config/%s", cluster.Name, cluster.BaseDNSDomain, models.HostRoleWorker)
```

Comment:
dont we have consts for ports?

Reply:
No, I didn't found any at least

Reply:
why api-int? it's does not have external dns - how would it work?

Reply:
it's always an ip. cannot be api-int